app/src/atoms/Banner/index.tsx

```diff
@@ -78,14 +78,18 @@ export function Banner(props: BannerProps): JSX.Element {
     isCloseActionLoading,
     padding,
     closeButton,
+    marginRight,
+    marginLeft,
+    size,
     ...styleProps
   } = props
   const bannerProps = BANNER_PROPS_BY_TYPE[type]
 
   const iconProps = {
     ...(icon ?? bannerProps.icon),
-    size: SPACING.spacing4,
-    marginRight: SPACING.spacing3,
+    size: size ?? SPACING.spacing4,
```

Comment:
The value is the same but using `SIZE_1` would be better for consistency.

```suggestion
    size: size ?? SIZE_1,
```

Reply:
line 118 
```ts
{onCloseClick && !isCloseActionLoading ? (
```

suggestion
```ts
{onCloseClick != null && !(isCloseActionLoading ?? false) ? (
```